BiweeklyMeetings/2022-07-05.md

```diff
@@ -0,0 +1,274 @@
+
+**Sebastien** -, and your opinion maybe, if you are here.
+
+**Sebastien** -:  I joined probably through the conversation, so I'm missing the beginning.
+
+**Matthias** - :  Yeah. Okay, sorry. We are talking about CIP 59, the terminologies surrounding co-features, which was proposed by Jared some time ago to clarify a bit of the terminology around the co-features. And I was saying, this one could probably just go as active right away because it's just explaining what's in there.
+
+**Sebastien** -:  Yeah, I think so. I think that makes sense. I read through it and everything, and it made sense to me. So I don't mind both merging and marking it is actually the right away.
+
+**Matthias** - :  We have a question. Oh sorry, that's some other stuff. So yeah okay, let's do that. I think we have reviewed it. I would just add my review now, that's all the... Yeah, All change has been made. Happy to approve. Happy to merge. Okay. Next one, on chain token metadata standard which has been reviewed for a while. We had a few remark questions. A lot of people were debating stuff in particular. The security of that, I think Michael Peyton Jones is usually against anything that is put on chain and it is based on minting transactions because indeed it has security issues or implications.
+
+**Matthias** - :  If we start heavily relying on people providing the metadata being the same as people minting the tokens, which has been generally the case in Cardano, but it's not a given. So I think he has a fair point. But yeah, the proposal has been eight all ever since, I think. It hasn't really moved, we are waiting for the author to come back to it. But yeah, I'm not sure if there is much to say on this one. Yeah. No remark, no question. I guess we can just leave it there.
+
+**Sebastien** -:  Yeah. I have something on this topic, but I'm going to have to address it in maybe a minute, if you just give me a minute.
+
+**Matthias** - :  Yeah, okay, okay. So depending on what you see, we'll see what to do with that one. Maybe it can also be picked up by someone else for a different standard or maybe reformulate the ID in a slightly different way. Yeah. And I don't know if there is any prior work also on that, I bet there is. So people putting all kind of metadata on chain. Now that has been the case for NFT, and I don't know if it has been the case for tokens themself or fungible tokens, at least I don't recall any project doing that.
```

Comment:
```suggestion
**Matthias** - :  Yeah, okay, okay. So depending on what you see, we'll see what to do with that one. Maybe it can also be picked up by someone else for a different standard or maybe reformulate the ID in a slightly different way. Yeah. And I don't know if there is any prior work also on that, I bet there is. So people putting all kind of metadata on chain. Now that has been the case for NFT, and I don't know if it has been the case for tokens themself -- I mean fungible tokens, at least I don't recall any project doing that.
```